vscode-visual-tests/features/analytics_extension_npm_cve_checks.feature

```diff
@@ -0,0 +1,366 @@
+Feature: Visual Studio Code + the Analytics plugin basic functionality: displaying problems on status bar
+
+    Scenario: Check that the Analytics plugin has been installed
+        Given The Visual Studio Code is set up
+         When I start the Visual Studio Code with parameter --list-extensions
+         Then I should find that extension redhat.fabric8-analytics is installed
+
+
+    Scenario: Analysis result for a NPM project with zero dependencies and thus zero CVEs
+        Given The PyAutoGUI library is initialized
+          And The Visual Studio Code is set up
+
+         # Start the Visual Studio Code
+         When I start the Visual Studio Code with parameter projects/npm_0_dependencies_0_CVEs/package.json
+         Then I should find Visual Studio Code instance
+
+         # Check the code area
+         When I wait 2 seconds
```

Comment:
I hope 2 seconds wait is enough to open.

Reply:
Yes, I thought about it. We might define several "wait-for" test steps, one for VSCode to start, one for the analysis etc. etc. Then the times won't be hardcoded and the retry will be part of the test step itself.

(2 seconds is more than enough in this case as we open just one file)